projects/bp-gallery/src/components/common/picture-gallery/PictureScrollGrid.tsx

```diff
@@ -70,6 +70,29 @@ const PictureScrollGrid = ({
     }
   }, [pictures, resultPictureCallback, loading]);
 
+  const maybeFetchMore = useCallback(() => {
+    if (loading) {
+      return;
+    }
+    let fetchCount = NUMBER_OF_PICTURES_LOADED_PER_FETCH;
+    if (maxNumPictures && pictures) {
+      fetchCount = Math.min(maxNumPictures - pictures.length, NUMBER_OF_PICTURES_LOADED_PER_FETCH);
+    }
+    if (fetchCount > 0) {
+      setIsFetching(true);
+      // @ts-ignore
+      // eslint-disable-next-line @typescript-eslint/no-unsafe-call
```

Comment:
why are you using ts-ignore and eslint disable here?

Reply:
I just refactored the code (you'll find it in the old code, too). But you are right, it's not needed and I removed it :)